src/modules/RA/DataLayer/components/tabs/MinisheetTab/index.js

```diff
@@ -50,7 +50,10 @@ const MinisheetTab = () => {
   const { geom_type: geomType, id: sourceId } = useSourceData('source');
 
   const { input: { value: fields } } = useField('fields');
-  const { input: { value: advanced } } = useField('minisheet_config.advanced', { initialValue: true });
+  const { input: { value: advanced } } = useField('minisheet_config.advanced', {
+    initialValue: true,
+    format: v => Boolean(v),
```

Comment:
Just thinking about it but
```jsx
    format: Boolean,
```
should work, doesn't it?

Reply:
Done.